### PR TITLE
Refactor the creation of instrument batchers to be in a central location

### PR DESCRIPTION
**Pull request by jkwatson**
This should allow us to build a Views API into the SDK.
Note: this PR is intended to be a pure internal refactoring, with no change in current functionality.

### CODE REVIEW

File: sdk/src/main/java/io/opentelemetry/sdk/metrics/Batchers.java [hunk 1]
```diff
@@ -34,36 +34,38 @@
...
-final class Batchers {
+public final class Batchers {
 
-  static Batcher getNoop() {
+  public static Batcher getNoop() {
     return Noop.INSTANCE;
   }
 
-  static Batcher getCumulativeAllLabels(
+  /** TODO: javadoc me. */
```
Comment (carlosalberto) [inline sdk/src/main/java/io/opentelemetry/sdk/metrics/Batchers.java:43]: Were these ones left empty on purpose?

Comment (jkwatson) [reply sdk/src/main/java/io/opentelemetry/sdk/metrics/Batchers.java:43]: nope! I'll fix that before merging.

Comment (jkwatson) [reply sdk/src/main/java/io/opentelemetry/sdk/metrics/Batchers.java]: done!

**Comment by jkwatson** [comment]
@carlosalberto FYI, I moved the ViewRegistry class up to the main `metrics` package, which cleaned up a bunch of visibility changes on other SDK internals.

**Pull request merged**